### DISCUSSION TITLE
Add text coloration

### DISCUSSION THREAD

**Discussion by AdoPi** [outline/outline#4815] [Ideas]
Hello,

It would be nice to have by some way a color picker for coloring some parts of a text.

What do you think?

**Comment by AdoPi** [comment]
Ok sorry, I'm not used to Github discussions and I find out I'm not the only one who talks about this feature. Could you please close my request?

**Comment by monir0908** [comment]
This is an essential feature that is overlooked. A drop-down menu with a pre-defined color range (5-10, perhaps) would be nice.
Glad you brought this up!
